server/src/main/java/umm3601/user/UserController.java

```diff
@@ -85,19 +83,19 @@ public void deleteUser(Context ctx) {
    */
   public void getUsers(Context ctx) {
 
-    List<Bson> filters = new ArrayList<Bson>(); // start with a blank document
+    List<Bson> filters = new ArrayList<>(); // start with a blank document
 
-    if (ctx.queryParamMap().containsKey("age")) {
-        int targetAge = ctx.queryParam("age", Integer.class).get();
-        filters.add(eq("age", targetAge));
+    if (ctx.queryParamMap().containsKey(AGE_KEY)) {
```

Comment:
SonarLint fussed about the string "company" appearing three times in `getUsers()`, which seemed like a reasonable complaint. So I converted the three key strings ("age", "company", and "role") into string constants.

One could argue that similar things can/should be done for some of the other string constants as well, but they're not repeated as much so I didn't do that here.

One could also argue that creating these constants is overkill here, so feel free to push back.